flexget/plugin.py

```diff
@@ -356,14 +356,10 @@ def _get_standard_plugins_path():
     :returns: List of directories where plugins should be tried to load from.
     """
     # Get basic path from environment
+    paths = []
     env_path = os.environ.get('FLEXGET_PLUGIN_PATH')
     if env_path:
-        # Get rid of trailing slashes, since Python can't handle them when
-        # it tries to import modules.
```

Comment:
No, I removed the stripping in this function. The trailing slashes were being removed again further down [L377](https://github.com/Ram-Z/Flexget/blob/plugin-path/flexget/plugin.py#L377)